packages/frontend-plugin-api/src/components/ExtensionBoundary.tsx

```diff
@@ -61,12 +65,12 @@ export function ExtensionBoundary(props: ExtensionBoundaryProps) {
   };
 
   return (
-    <ExtensionSuspense>
+    <Suspense fallback="Loading...">
```

Comment:
Using application components here causes an infinite loop in tests, any clues?

Reply:
I have the feeling that this entire component should be the new default ErrorBoundary core component extension.

Reply:
Probably the progress fallback right? Related to the discussion on that above

Reply:
Maybe. Or, could be good to separate them apart perhaps so you don't have to reimplement everything (analytics etc etc) just to tweak the error presentation? Not sure, I don't have a strong opinion. 🤔 Do we want to compose these or not, essentially

Reply:
https://github.com/backstage/backstage/pull/21652